### PR TITLE
Commands tests

### PR DESCRIPTION
**Pull request by youaremysky99**
- All the tests passed

- Check-style okay.

### CODE REVIEW

File: src/test/java/seedu/fma/logic/commands/AddCommandIntegrationTest.java [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
+
+    private Model model;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalLogBook(), new UserPrefs());
+    }
+
+    @Test
+    public void execute_newPerson_success() {
```
Comment (Jillzyt) [inline src/test/java/seedu/fma/logic/commands/AddCommandIntegrationTest.java]: Remember to change the method name to match Logs.

File: src/test/java/seedu/fma/logic/commands/AddCommandIntegrationTest.java [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
+
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        expectedModel.addLog(validLog);
+
+        assertCommandSuccess(new AddCommand(validLog), model,
+                String.format(AddCommand.MESSAGE_SUCCESS, validLog), expectedModel);
+    }
+
+    @Test
+    public void execute_duplicatePerson_throwsCommandException() {
```
Comment (Jillzyt) [inline src/test/java/seedu/fma/logic/commands/AddCommandIntegrationTest.java]: Remember to change the method name to match Logs.

File: src/test/java/seedu/fma/logic/commands/FindCommandTest.java [hunk 3]
```diff
@@ -1,83 +1,78 @@
...
+        String expectedMessage = String.format(MESSAGE_LOGS_LISTED_OVERVIEW, 0);
+        NameContainsKeywordsPredicate predicate = preparePredicate(" ");
+        FindCommand command = new FindCommand(predicate);
+        expectedModel.updateFilteredLogList(predicate);
+        assertCommandSuccess(command, model, expectedMessage, expectedModel);
+        assertEquals(Collections.emptyList(), model.getFilteredLogList());
+    }
+
+    @Test
+    public void execute_multipleKeywords_multiplePersonsFound() {
```
Comment (Jillzyt) [inline src/test/java/seedu/fma/logic/commands/FindCommandTest.java]: Remember to change the method name to match Logs. This is repeated for the other tests in this file

**Review by Jillzyt — CHANGES_REQUESTED** [review]
LGTM 👍  Is the few instances of person left intentionally?

**Comment by jeminsieow** [comment]
Remember to pull and fix the merge conflicts

**Comment by Jillzyt** [comment]
LGTM.

**Review by Jillzyt — APPROVED** [review]

**Pull request merged**